Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
@thinkyhead - I answered on the PR, but yeah: compile sizes were identical for PBrevF (1286) and BRAINWAVE (647).  I didn't try any others.